ldap-connector/config/variables.yaml

```diff
@@ -6,4 +6,7 @@
 # '<project>/Config/_<environment>/variables.yaml
 #
 Variables:
-#  myVariable: value
+  LdapConnector:
+    Username: ""
+    Password: ""
+    Url: "ldap://test-ad.ivyteam.io:389"
```

Comment:
this is the variables yaml of your real product.
put test data in variables.yaml of your test project.
the variables will be overridden.

the URL here must be empty
and you can URL in your test project has the test url.